debian/patches/01_makefile_fixes.patch

```diff
@@ -16,7 +16,7 @@ This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
 -CC=gcc
 -CFLAGS=-O2 -DNXT	# on some systems add: -I/usr/include
 +CC?=gcc
-+CFLAGS?=-O2 -DNXT	# on some systems add: -I/usr/include
++CFLAGS+=-O2 -DNXT	# on some systems add: -I/usr/include
```

Comment:
Totally agree but to be clear as it kind of gets lost in the diff-of-a-diff noise: the only real change here is the `CFLAGS` line changing from `CFLAGS?=` to `CFLAGS+=`. The `CC?=` change etc. landed in an earlier commit.